nav_msgs/Path path_msg
---
bool call_back_msg
